src/pages/users/AddUsers.vue

```diff
@@ -913,7 +920,7 @@ const getOrgId = async (orgType, orgName, parentDistrict = ref(null), parentScho
     }
   }
 
-  orgIds[orgType][normalizedOrgName] = orgs[0].id;
+  orgIds[orgType][cacheKey] = orgs[0].id;
```

Comment:
Could you explain how this change fixes the issue? I'm a little bit confused how changing the index from normalizedOrgName to the cacheKey fixes the issue.

Reply:
Got it. Thanks.